include/splinterdb/splinterdb.h

```diff
@@ -60,7 +60,8 @@ typedef struct splinterdb_config {
 
    // Shared memory support
    uint64 shmem_size;
-   _Bool  use_shmem; // Default is FALSE.
+   _Bool  use_shmem;  // Default is FALSE.
+   _Bool  fork_child; // Default is FALSE
```

Comment:
re-visit -- whether this field is needed any more in this global struct. Delete, if not needed.

Reply:
Good catch, @rtjohnso - yes, this can come out. Deleted.